src/app/(nextjs_migration)/(guest)/page.tsx

```diff
@@ -14,7 +17,12 @@ import LockImage from "../../../client/images/landing-lock@2x.webp";
 import MailImage from "../../../client/images/landing-mail@2x.webp";
 import NaturePhoneImage from "../../../client/images/landing-nature-phone@2x.webp";
 
-export default function Home() {
+export default async function Home() {
+  const session = await getServerSession(authOptions);
+  if (session) {
+    redirect("/user/breaches");
+  }
```

Comment:
Wasn't it intentional in the current website that we wouldn't redirect use to the dashboard from the landing page, even if they're logged in?

Reply:
Right, I’ve got it the other way around with the pre-refresh site: [5f2509a](https://github.com/mozilla/blurts-server/pull/3058/commits/5f2509a3a8cd476db1787aa013f06c981626c133).

Reply:
Ah, I didn't realise you'd have to specify a redirect URL when calling `signIn`, but nice, that works well.